<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Биржа труда</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Список кандидатов</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="document-open">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="document-save">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="Line" name="line_2">
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="midLineWidth">
             <number>5</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="list-add">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="list-remove">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_7">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="accessories-text-editor">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Заполните поля (не используйте запятые пжпж!)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ФИО:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_23">
            <item>
             <widget class="QComboBox" name="comboBox"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_3"/>
            </item>
           </layout>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Дата рождения:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QDateEdit" name="dateEdit">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Специальность:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="lineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Высшее образование:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QCheckBox" name="checkBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Желаемая должность</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Минимальный оклад</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QSpinBox" name="spinBox">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>50000</number>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>Отменить изменения</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_9">
              <property name="text">
               <string>Сохранить изменения</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="12" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>Отмена</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Ок</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="13" column="0" colspan="2">
           <widget class="Line" name="line">
            <property name="midLineWidth">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="14" column="1">
           <widget class="QTextEdit" name="textEdit">
            <property name="undoRedoEnabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="16" column="1">
           <widget class="QTextEdit" name="textEdit_2">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="15" column="0" colspan="2">
           <widget class="Line" name="line_4">
            <property name="midLineWidth">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="17" column="1">
           <widget class="QTextEdit" name="textEdit_3">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="14" column="0">
           <widget class="QPushButton" name="pushButton_19">
            <property name="text">
             <string>Список вакансий</string>
            </property>
           </widget>
          </item>
          <item row="16" column="0">
           <widget class="QPushButton" name="pushButton_20">
            <property name="text">
             <string>Список дефицитных
вакансий</string>
            </property>
           </widget>
          </item>
          <item row="17" column="0">
           <widget class="QPushButton" name="pushButton_21">
            <property name="text">
             <string>Список фирм, для
которых не найдены
кандидаты</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Список фирм</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QPushButton" name="pushButton_10">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="document-open">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_11">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="document-save">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_13"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="Line" name="line_3">
            <property name="midLineWidth">
             <number>6</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QPushButton" name="pushButton_13">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="list-add">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_14">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="list-remove">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_12">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="accessories-text-editor">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Заполните поля (не используйте запятые пжпж):</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Фирма:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QComboBox" name="comboBox_2"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_4"/>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Специальность:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_5"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Должность:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit_6"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Оклад:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="spinBox_2">
            <property name="maximum">
             <number>100000</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Дни отпуска:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="spinBox_3">
            <property name="maximum">
             <number>365</number>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Высшее образование:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Минимальный возраст:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QSpinBox" name="spinBox_4"/>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Максимальный возраст:</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QSpinBox" name="spinBox_5">
            <property name="maximum">
             <number>120</number>
            </property>
           </widget>
          </item>
          <item row="11" column="0" colspan="2">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_16">
              <property name="text">
               <string>Ок</string>
              </property>
              <property name="icon">
               <iconset theme="face-sad"/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_15">
              <property name="text">
               <string>Сохранить изменения</string>
              </property>
              <property name="icon">
               <iconset theme="emblem-system"/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_17">
              <property name="text">
               <string>Отменить изменения</string>
              </property>
              <property name="icon">
               <iconset theme="emblem-symbolic-link"/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_18">
              <property name="text">
               <string>Отменить</string>
              </property>
              <property name="icon">
               <iconset theme="face-kiss"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
